### PR TITLE
requestPermission function, add onPermissionDefaultCallback

### PR DESCRIPTION
**Pull request by zhenshuaiwws**
I need this callback to control page show tip

### CODE REVIEW

**Comment by alexgibson** [comment]
You need to modify the source, not the file in the /dist folder. Can you also please explain why you need this a bit more? It seems a bit confusing to call the permission denied callback on the default state.

**Comment by zhenshuaiwws** [comment]
Because I use the angular package, the need to maintain the callback does "not set" the state

**Comment by zhenshuaiwws** [comment]
Like slack prompts the user "not set" or "disabled"

File: dist/notify.js [hunk 1]
```diff
@@ -202,4 +207,6 @@
 
   exports['default'] = Notify;
 
+  Object.defineProperty(exports, '__esModule', { value: true });
```
Comment (alexgibson) [inline dist/notify.js]: Are you using a different version of jspm to the version that's in `package.json` here? When I pull down your branch locally and run `npm run build` this doesn't seem to get generated.

Comment (alexgibson) [inline dist/notify.js]: Yep, please also update the dev dependency in `package.json` here to use the latest version of `jspm@beta` beta.

**Comment by alexgibson** [comment]
Closing due to inactivity

**Pull request closed without merging**